src/dotty/tools/dotc/Compiler.scala

```diff
@@ -60,6 +60,8 @@ class Compiler {
            new ElimByName,
            new ResolveSuper),
       List(new Erasure),
+      List(new ElimErasedValueType),
```

Comment:
Is there a reason you need separate phases for those? Cannot they be blocked together with Mixin?